terraform-e2e/README.md

```diff
@@ -26,23 +28,6 @@ terraform import module.en.google_app_engine_application.app ${PROJECT_ID}
 terraform import module.en.google_firebase_project.default ${PROJECT_ID}
 ```
 
-#### Troubleshooting: `terraform import` failure
-
-If you ran `terraform import` and see the error of invalid provider configuration like:
-
-```text
-Error: Invalid provider configuration
-
-  on ../terraform/main.tf line 15:
-  15: provider "google" {
-
-The configuration for
-module.en.provider["registry.terraform.io/hashicorp/google"] depends on values
-that cannot be determined until apply.
-```
-
-This can be mitigated by commenting out [project = var.project](https://github.com/google/exposure-notifications-verification-server/blob/86a05f09dabfeddbce73c9d628ec01d3738782ed/terraform/main.tf#L16), as well as [project = var.project](https://github.com/google/exposure-notifications-verification-server/blob/86a05f09dabfeddbce73c9d628ec01d3738782ed/terraform/main.tf#L23). This is similar to the issue described in https://github.com/hashicorp/terraform/issues/25816, and might be fixed in 0.13.1 release of terraform.
```

Comment:
This had been confirmed fixed in terraform 1.13.1

Reply:
I'd rather leave this in with a note at the top saying "upgrade to the latest Terraform version first"

Reply:
Done